specs/pvs/corla.pvs

```diff
@@ -243,8 +258,10 @@ BEGIN
 %%% unique name.
 % @review CDOS will there be one risk limit? If more than one, then
 % system must provide a way to map contests to risk limits.
```

Comment:
There will be not one risk limit but some number of (Contest, risk limit) pairs.